uberfire-extensions/uberfire-widgets/uberfire-widgets-commons/src/main/java/org/uberfire/ext/widgets/common/client/tables/SimpleTable.java

```diff
@@ -112,7 +124,7 @@ public void setGridPreferencesStore(final GridPreferencesStore gridPreferences)
 
     public void saveGridPreferences() {
         if (gridPreferencesStore != null && preferencesService != null) {
-            gridPreferencesStore.setPreferenceKey(gridPreferencesStore.getGlobalPreferences().getKey());
+            gridPreferencesStore.setPreferenceKey(gridPreferencesStore.getPreferenceKey());
```

Comment:
@cristianonicolai  The change introduced at SimpleTable savePreferences is to use that PreferenceKey instead that globalPreferences().getKey(). This allow to remove the proposed code to maintain both keys aligned using just the PreferenceKey that could be changed externally

Reply:
my only concern with this is potentially breaking backwards compatibility as this component is used in many places. Could we do this on jbpm-wb only? That way would be more under control.

Reply:
@cristianonicolai updated, I split the saveGridPreferences to keep the same behaviour using the gridGlobalPreference key but allowing to save the preferences directly using the PreferenceKey.